code/game/turfs/open.dm

```diff
@@ -4,7 +4,25 @@
 	var/wet = 0
 	var/wet_time = 0 // Time in seconds that this floor will be wet for.
 	var/image/wet_overlay = null
-
+	var/already_frozen = FALSE
+	var/freeze_timer = 0
+	var/freeze_max = 3
+
+/turf/open/freon_gas_act()
+	if(!already_frozen)
+		for(var/obj/I in contents)
+			if(!I.is_frozen)
+				make_frozen_visual(I)
+		already_frozen = TRUE
+	freeze_timer++
+	if(freeze_timer >= freeze_max)
+		freeze_timer = 0
+		for(var/obj/IT in contents)
+			if(!IT.is_frozen)
+				make_frozen_visual(IT)
+	MakeSlippery(TURF_WET_PERMAFROST)
+	air.temperature = 2.7
```

Comment:
This is so wrong, so very wrong. Just have the gas have a high specific heat and start out cold. Otherwise even the tiniest trace of this will keep a room cold forever unless you space the entire room and rebuild it. A station flood of this would be an instant shuttle call.